tests/test_subset.py

```diff
@@ -1,16 +1,47 @@
 import os
 
-import geopandas as gpd
 import numpy as np
 import pytest
 import xarray as xr
 
-from xclim import subset
+# class TestSubsetImport:
+#     subset = pytest.importorskip('clisops.core.subset', reason="`clisops` subset utilities are not installed.")
+#     pytest.importorskip("rtree", reason="rtree spatial indexing utilities are not installed.")
+
+try:
+    import clisops.core.subset as subset
+    import geopandas as gpd
+except ImportError:
+    subset = False
+    gpd = False
```

Comment:
```suggestion
try:
    import clisops.core.subset as subset
    import geopandas as gpd
except ImportError as e:
    pytestmarkskip_gis = pytest.mark.skip(reason=str(e))
```

Reply:
This is raising errors as it changes a lot of the logic behind skipping tests peppered throughout the file. This also causes a problem when clisops is installed as `pytestmarkskip_gis` is undefined. I imagine an `else` statement would work but I'm not sure what to define that decorator as in the case that clisops is found.